### ISSUE TITLE
project description

### ISSUE THREAD

**Issue by petrus-jvrensburg** [Schultzer/cldr_territories#30]
I know this is really nitpicking, but the project description is currently pointing at https://github.com/kipcole9/cldr when it should point to https://github.com/elixir-cldr/cldr.

**Comment by Schultzer** [comment]
> I know this is really nitpicking, but the project description is currently pointing at https://github.com/kipcole9/cldr when it should point to https://github.com/elixir-cldr/cldr.

I'm intrigued to hear your reasoning for this change; I assume you are familiar with this and the other repositories' history.

Currently, three repositories share the same link in their description.

- https://github.com/elixir-cldr/cldr_lists
- https://github.com/elixir-cldr/cldr_dates_times
- https://github.com/Schultzer/cldr_territories

**Comment by kipcole9** [comment]
@petrus-jvrensburg , @Schultzer, forgive my confusion. I can't see the reference to `/kipcole9` in `cldr_territories`. And equally I can't find in `cldr_dates_times` or `cldr_lists`.  I did find a couple of outdated links to the wrong repo in `cldr_currencies` and `cldr` and I fixed both of those just now.

Would someone mind pasting a link to where the repo is referenced as `https://github.com/kipcole9` rather than the correct (for my libs that is) `https://github.com/elixir-cldr` ?

**Comment by Schultzer** [comment]
> @petrus-jvrensburg , @Schultzer, forgive my confusion. I can't see the reference to `/kipcole9` in `cldr_territories`. And equally I can't find in `cldr_dates_times` or `cldr_lists`. I did find a couple of outdated links to the wrong repo in `cldr_currencies` and `cldr` and I fixed both of those just now.
> 
> Would someone mind pasting a link to where the repo is referenced as `https://github.com/kipcole9` rather than the correct (for my libs that is) `https://github.com/elixir-cldr` ?

I don’t think this issue is regarding any links in any documentation or code, but rather the Github description of the repositories, currently only three repositories contains a link, two for them pointing to ‘https://github.com/kipcole9’ and one to ‘ https://github.com/elixir-cldr’.

The majority does not contain any links.

For what it’s worth, I think streamlining the repositories description is a good idea.

**Comment by kipcole9** [comment]
Ahhhhhh that makes sense!  Will revisit all of mine today.Maybe worth placing cldr_territories in the same GitHub Organization?  Your copyright still of course, and your ownership of the repo. But maybe easier discover for people if they are all in the same organisation?Sent from my iPhoneOn 8 Jul 2023, at 07:37, Benjamin Schultzer ***@***.***> wrote:﻿

@petrus-jvrensburg , @Schultzer, forgive my confusion. I can't see the reference to /kipcole9 in cldr_territories. And equally I can't find in cldr_dates_times or cldr_lists. I did find a couple of outdated links to the wrong repo in cldr_currencies and cldr and I fixed both of those just now.
Would someone mind pasting a link to where the repo is referenced as https://github.com/kipcole9 rather than the correct (for my libs that is) https://github.com/elixir-cldr ?

I don’t think this issue is regarding any links in any documentation or code, but rather the Github description of the repositories, currently only three repositories contains a link, two for them pointing to ‘https://github.com/kipcole9’ and one to ‘ https://github.com/elixir-cldr’.
The majority does not contain any links.
For what it’s worth, I think streamlining the repositories description is a good idea.

**Comment by Schultzer** [comment]
> Ahhhhhh that makes sense!  Will revisit all of mine today.Maybe worth placing cldr_territories in the same GitHub Organization?  Your copyright still of course, and your ownership of the repo. But maybe easier discover for people if they are all in the same organisation?

I think it would benefit the community if all the ex_cldr libraries are in the same organization.

As of now cldr_territories and cldr_languages are forks.

Keeping everything under one organization could potentially help with streamlining description, CI, CHANGELOG, README and coordinating releases with GitHub Actions.

I wonder if we should move this to a discussion in the organization instead of here and have @LostKobrakai being a part of it.
